libs/api/src/services/admin/index.ts

```diff
@@ -11,15 +7,27 @@ import { ErrorsResponse, ErrorsResponseSchema, OkResponse } from '../../base';
  * @method POST
  */
 export interface PostCvrsRequest {
-  files?: any;
+  signature: string;
+  name: string;
+  precinctIds: string[];
+  scannerIds: string[];
+  timestamp: string;
+  // TODO: this should not be any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  castVoteRecords?: any;
```

Comment:
Could we make this `unknown` instead for now? Or does that require too much updating / casting elsewhere?

Reply:
What is the actual type? Can we just use `CastVoteRecord`?

Reply:
Not clear to me immediately how to write a schema for `CastVoteRecord` and I want to make sure the `nh-demo` branch is at minimum functional in `main` before I'm OOO, so I filed a follow-up ticket #2168 to use the correct type for `PostCvrsRequestSchema` as well.

Reply:
Okay, let's do that in a follow-up.